### PR TITLE
[FIX] Verify if keyword needs indentation for all languages

### LINKED ISSUE #2007: indentation needs to be checked for all languages
Related to #1213 

Here we only check for English keywords:
https://github.com/Felienne/hedy/blob/ae64eb74face0531bbe89068f1308806af42701d/hedy.py#L1951

We could at use the translation code that knows the equivalent of these for all langs.

### PR DESCRIPTION
**Pull request by TiBiBa**
**Description**
Currently we use a fixed list to check if a keywords needs indentation on the next line. This doesn't make sense anymore as keywords are generated dynamically based on the translations provided by contributors. In this PR we fix the static check function by making it dependent on a dynamically created list containing the keywords that need indentation for all supported keyword languages.

**Fixes**
This PR fixes #2007.

**How to test**
Nothing to test, if the tests pass it is ready to merge!

### CODE REVIEW

File: hedy.py [hunk 1]
```diff
@@ -33,6 +35,12 @@
 # Python keywords need hashing when used as var names
 reserved_words = ['and', 'except', 'lambda', 'with', 'as', 'finally', 'nonlocal', 'while', 'assert', 'False', 'None', 'yield', 'break', 'for', 'not', 'class', 'from', 'or', 'continue', 'global', 'pass', 'def', 'if', 'raise', 'del', 'import', 'return', 'elif', 'in', 'True', 'else', 'is', 'try']
 
+# Let's retrieve all keywords dynamically from the cached KEYWORDS dictionary
+indent_keywords = []
+for lang, keywords in KEYWORDS.items():
+    for keyword in ['if', 'for', 'repeat']:
```
Comment (Felienne) [inline hedy.py:41]: We might miss `while` here (I know we did not have it in the original but I think that was a mistake!)

Comment (TiBiBa) [reply hedy.py:41]: Whoops! Will add.

Comment (Felienne) [reply hedy.py:41]: So that was my bad, originally :)

**Review by Felienne — APPROVED** [review]
Turns out we missed else too! Added it with tests.

**Pull request merged**